tools/m2kcli/commands/command_in.cpp

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2019 Analog Devices, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+
+#include <iomanip>
+#include "command_in.h"
+
+using namespace libm2k::cli;
+
+void CommandIn::printSamplesCsvFormat(std::vector<uint16_t> &samples)
+{
+	for (unsigned short &sample : samples) {
+		std::cout << sample << std::endl;
+	}
+}
+
+void CommandIn::printSamplesCsvFormat(std::vector<std::vector<double>> &samples)
+{
+	int nbChannels = samples.size();
```

Comment:
maybe just do `int nbChannels = samples.size() - 1;` here; then you don't need to use `nbChannels - 1` below

Reply:
The modification was added in commit 52b4501.